### DISCUSSION TITLE
Would like to ask for suggestion on how to delete Google Analytics Cookie after user reject approval

### DISCUSSION THREAD

**Discussion by stanislav-kukucka** [orestbida/cookieconsent#628] [Q&A]
Here is my link to [my config](https://wallcreo.sk/cookieconsent-config.js).

**Comment by orestbida** [comment]
Hi @stanislav-kukucka,

cookies are deleted by the plugin. What you need to do is tell ga4 to stop the tracking (and not recreate the cookies).
You can do this by setting `window['ga-disable-G-XXXXXXXXXX'] = true;`

```js

services: {
    ga4: {
        label: 'Google Analytics 4',
        onAccept: () => {
            window.dataLayer = window.dataLayer || [];
            window.gtag = function(){dataLayer.push(arguments);}
            gtag('js', new Date());
            gtag('config', 'G-XXXXXXXXXX');
    
            CookieConsent.loadScript('https://www.googletagmanager.com/gtag/js?id=G-XXXXXXXXXX');
        },
        onReject: () => {
            window['ga-disable-G-XXXXXXXXXX'] = true;
        },
        cookies: [
            {
                name: /^(_ga|_gid)/
            }
        ]
    }
}

**Reply by yemling** [reply]
I'm trying to figure this out too. 
Does window['ga-disable-G-XXXXXXXXXX'] = true; actually delete the cookie?

**Reply by orestbida** [reply]
@yemling, It does not delete cookies, but it [disables Google Analytics tracking](https://developers.google.com/tag-platform/security/guides/privacy) (according to google).
